src/tests/demo-flow-test.ts

```diff
@@ -11,7 +11,7 @@
 import {assert} from '../platform/chai-web.js';
 import {Loader} from '../platform/loader.js';
 import {Manifest} from '../runtime/manifest.js';
-import {PlanningTestHelper} from '../planning/testing/arcs-planning-testing.js';
+import {PlanningTestHelper} from '../planning/testing/planning-test-helper.js';
```

Comment:
We're thinking this file can probably just be deleted - the demo flow test hasn't been enabled for a long time (and the other one is a super-trivial load test). We have that wdio tests for some reasonable level of integration coverage.

Reply:
i'll make a separate pr to remove it.